src/Symfony/Bridge/Twig/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+5.3.0
+-----
+
+ * added support for Bootstrap 5
```

Comment:
we changed our convention for changelogs, here is how this should look like now (we don't update previous lines):
```
5.3
---

 * Add support for Bootstrap 5
```

Reply:
Fixed